optimizely/optimizely_config.py

```diff
@@ -64,41 +81,43 @@ def __init__(self, id, key, experiments_map, variables_map):
         self.experiments_map = experiments_map
 
         self.variables_map = variables_map
-        self.delivery_rules = []
-        self.experiment_rules = []
+        self.delivery_rules: list[OptimizelyExperiment] = []
+        self.experiment_rules: list[OptimizelyExperiment] = []
 
 
 class OptimizelyVariation:
-    def __init__(self, id, key, feature_enabled, variables_map):
+    def __init__(
+        self, id: str, key: str, feature_enabled: Optional[bool], variables_map: dict[str, OptimizelyVariable]
```

Comment:
is feature_enabled optional?

Reply:
It looks like it. On line 367, the default option is `None` and not a `bool`.
https://github.com/optimizely/python-sdk/blob/9358e1be32b81e4e45d62c6168efad372e932206/optimizely/optimizely_config.py#L365-L371